geomesa-hbase/geomesa-hbase-datastore/src/main/scala/org/locationtech/geomesa/hbase/coprocessor/aggregators/HBaseBinAggregator.scala

```diff
@@ -9,24 +9,20 @@
 package org.locationtech.geomesa.hbase.coprocessor.aggregators
 import org.geotools.factory.Hints
 import org.locationtech.geomesa.features.ScalaSimpleFeature
-import org.locationtech.geomesa.filter.function.BinaryOutputEncoder
-import org.locationtech.geomesa.filter.function.BinaryOutputEncoder.BIN_ATTRIBUTE_INDEX
 import org.locationtech.geomesa.hbase.HBaseFeatureIndexType
 import org.locationtech.geomesa.hbase.coprocessor.GeoMesaCoprocessor
 import org.locationtech.geomesa.index.iterators.{BinAggregatingScan, ByteBufferResult}
+import org.locationtech.geomesa.utils.bin.BinaryOutputEncoder
 import org.locationtech.geomesa.utils.geotools.GeometryUtils
 import org.opengis.feature.simple.{SimpleFeature, SimpleFeatureType}
 import org.opengis.filter.Filter
 
 class HBaseBinAggregator extends BinAggregatingScan with HBaseAggregator[ByteBufferResult]
 
 object HBaseBinAggregator {
-  def bytesToFeatures(bytes : Array[Byte]): SimpleFeature = {
-    val sf = new ScalaSimpleFeature("", BinaryOutputEncoder.BinEncodedSft)
-    sf.setAttribute(1, GeometryUtils.zeroPoint)
-    sf.setAttribute(BIN_ATTRIBUTE_INDEX, bytes)
-    sf
-  }
+
+  def bytesToFeatures(bytes : Array[Byte]): SimpleFeature =
+    new ScalaSimpleFeature("", BinaryOutputEncoder.BinEncodedSft, Array(bytes, GeometryUtils.zeroPoint))
```

Comment:
Nice.  I don't know if this PR is the right place for it, but did you look for other places where this could be done?

One +/- here is that BIN_ATTRIBUTE_INDEX isn't used explicitly.  If the BIN_SFT is changed, we may not know to change this line...